dts/bindings/fuel-gauge/battery_cutoff.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2023 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+
+  Properties for fuel-gauges that may control battery cutoff, this is common in SBS compliant or
+  similarly smart battery fuel gauges.
+
+  Note: These properties are to be used with meaningful defaults in fuel gauge ICs that implement
+  battery cutoff support. See the default fuel gauge SBS Gauge compatible as an example.
+
+properties:
+  battery-cutoff-support:
+    description: |
+      Helper prop that indicates whether this device can cutoff the battery, this is also often
+      referred to as ship or sleep mode. Any non-zero value means battery cutoff is supported.
+    type: boolean
+  battery-cutoff-reg-addr:
+    description: |
+       Address of register to receive cutoff payload for battery cutoff.
+    type: int
+  battery-cutoff-payload:
+    description: |
+       Payload to write to cutoff battery register. This must be array of 2 integers. If your
+       battery only requires a single payload, duplicate the value.  This will cause the driver to
+       issue the cutoff twice.
```

Comment:
Looking closer at your macros, you actually support a payload length of 1 or 2.  Change this help to note you can specify a maximum of 2 integers and drop the text about setting a duplicate value.

Reply:
Yeah, though I was fine with being restrictive at first and just relaxing that as needed.